azure-pipelines/ui-automation/msal-local-nightly.yml

```diff
@@ -0,0 +1,159 @@
+# run MSAL UI automation testcases
+# Variable: 'ENV_VSTS_MVN_ANDROIDADACCOUNTS_USERNAME' was defined in the Variables tab
+# Variable: 'gCloudProjectId' was defined in the Variables tab
+# Variable: 'mvnAccessToken' was defined in the Variables tab
+name: $(Build.BuildId)_$(Build.DefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+trigger: none
+pr: none
+
+schedules:
+  - cron: "0 5 * * 1-6" # 5:00 AM UTC everyday Mon-Sat
+    displayName: Auth Client Android SDK dev build
+    branches:
+      include:
+        - dev
+    always: true
+
+variables:
+  engineeringProjectId: 'fac9d424-53d2-45c0-91b5-ef6ba7a6bf26'
+  azureSamplePipelineId: 1458
+  msalAutomationAppApk: msalautomationapp-local-BrokerHost-debug.apk
+  msalAutomationAppTestApk: msalautomationapp-local-BrokerHost-debug-androidTest.apk
+  azureSampleApk: AzureSample-local-debug.apk
+  firebaseTimeout: 45m
+  resultsHistoryName: Dev MSAL without Broker
+
+parameters:
+  - name: firebaseDeviceId
+    displayName: Firebase Device Id
+    type: string
+    default: blueline
+  - name: firebaseDeviceAndroidVersion
+    displayName: Firebase Device Android Version
+    type: number
+    default: 28
+
+stages:
+  # msalautomationapp
+  - stage: 'msalautomationapp'
+    displayName: Build MSAL Automation APKs
+    jobs:
+      - job: 'msalautomationapp'
+        displayName: Build and Publish MSAL Automation app
+        pool:
+          vmImage: ubuntu-latest
+        steps:
+          - checkout: self
+            clean: true
+            submodules: recursive
+            persistCredentials: True
+          - bash: |
+              echo "##vso[task.setvariable variable=ENV_VSTS_MVN_ANDROID_MSAL_ACCESSTOKEN]$(mvnAccessToken)"
+            displayName: Set MVN Access Token in Environment
+          - task: AzureKeyVault@2
+            displayName: 'Get Key vault AndroidAutomationRunnerAppSecret'
+            inputs:
+              azureSubscription: 'MSIDLABS_ANDROID_KV'
+              KeyVaultName: 'ADALTestInfo'
+              SecretsFilter: 'AndroidAutomationRunnerAppSecret'
+              RunAsPreJob: false
+          - task: Gradle@2
+            displayName: 'Assemble MSAL Automation App'
+            inputs:
+              tasks: clean msalautomationapp:assembleLocalBrokerHostDebug -PlabSecret=$(AndroidAutomationRunnerAppSecret) -PbrokerSource=LocalApk
```

Comment:
this should be PlayStore, right? 
Does it matter if we don't run broker tests here?

Reply:
It doesn't matter. I think I can remove passing this param here at all. It would default to PlayStore, but again it would be ignored as it doesn't matter.

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1644/commits/3a6e2a69ddd181aeaaf447b69409889af270c13c